server/raft_transport.go

```diff
@@ -32,8 +33,19 @@ import (
 const (
 	raftServiceName = "MultiRaft"
 	raftMessageName = raftServiceName + ".RaftMessage"
+	// Outgoing messages are queued on a per-node basis on a channel of
+	// this size.
+	raftSendBufferSize = 500
+	// When no message has been sent to a Node for that duration, the
+	// corresponding instance of processQueue will shut down.
+	raftIdleTimeout = time.Minute
 )
 
+// raftMessageTimeout specifies the acceptable wait time for a single
+// message to a node to succeed.
+// It is a variable to adapt it for race testing on slow machines.
+var raftMessageTimeout = 1 * time.Second
```

Comment:
1 second seems aggressive for production too in geographically-distributed deployments. Raft messages may be large (snapshots) and since snapshots are often the first things sent the TCP connection may still be in its slow-start mode.